src/screens/WelcomeScreen.tsx

```diff
@@ -0,0 +1,93 @@
+import React from 'react';
+import { useFonts } from 'expo-font';
+import ButtonComponent from '../components/ButtonComponent';
+import { View, StyleSheet, Image, Text } from 'react-native';
+
+const WelcomeScreen = ({ navigation }: { navigation: any }) => {
+  const [fontsLoaded] = useFonts({
+    'Roboto-Medium': require('../../assets/fonts/Roboto/Roboto-Medium.ttf'),
+    'Roboto-Regular': require('../../assets/fonts/Roboto/Roboto-Regular.ttf'),
+    'NotoSerif-Bold': require('../../assets/fonts/NotoSerif/NotoSerif-Bold.ttf'),
+    'Inter-Regular': require('../../assets/fonts/Inter/Inter-Regular.ttf'),
+  });
+
+  if (!fontsLoaded) {
+    return <Text>Loading...</Text>;
+  }
+
+  function handleLogin(): void {
+    navigation.navigate('LoginScreen');
+  }
+
+  function handleSignUp(): void {
+    navigation.navigate('SignUpScreen');
+  }
+
+  return (
+    <View style={styles.container}>
+      <Image
+        source={{
+          uri: 'https://s3-alpha-sig.figma.com/img/ceba/f0c9/1d52df65bfda2146d8a56199f05e993c?Expires=1716768000&Key-Pair-Id=APKAQ4GOSFWCVNEHN3O4&Signature=PbvjzmBR0H1OEIP7kYo8OhAQSI9RAQh9XaiXD5-6PQFNEONVvgAT7j2FZoDX5LV-Ni0jhpLQe2Fb1Igm0Trz5uPsPNcFVpj3C4JvWF3sGYOVuWvHD50bsJt8EwMAugCSWSiWLiW4WXyRuSO7jvgePgeBHytcafC1byPQqRLxIUiAUpOjyqPq1yJhk1Ax6qzKrs9G5zLY11bY2MsevVSvI5tIWrHhMSsz2W2kP61elVfkbmuMHAQTaHMHUvCtWjxifMdcJiTIblt~356FjxQKgSSsuuWsNHOiA49Q2yFf~HgLQ-f1XQ8F4kX5p7lI3UiiFE6nk6jP15iZqfHk8pMWPw__',
```

Comment:
why point to a remote URI? unless we're memory constrained, it would probably be better to have the image file in the assets folder.

Reply:
I am waiting for Monica to share the google drive of pics. Once I get that I will change them.

Reply:
Just as a note for future, exposing project URLs like this in code is bad practice, ask for placeholder assets or use an environment variable